packages/core/src/App/Components/Layout/Footer/live-chat.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Popover, Icon } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import BinarySocket from '_common/base/socket_base';
+// import { WS } from 'Services';
+import { connect } from 'Stores/connect';
+
+const LiveChat = ({ email, is_logged_in, loginid, landing_company_shortcode, currency, residence }) => {
+    const [is_livechat_interactive, setLiveChatInteractive] = React.useState(false);
+
+    React.useEffect(() => {
+        if (window.LiveChatWidget) {
+            BinarySocket.wait('get_settings').then(response => {
+                const get_settings = response.get_settings || {};
+                const { first_name, last_name } = get_settings;
```

Comment:
authorize call has a field for full name, can we use that instead?

Reply:
@negar-binary I think this is nicer? since it's without the title `Mr John Doe` it will be `John Doe`

Reply:
okay, sure